certified-connectors/RescoReports/apiDefinition.swagger.json

```diff
@@ -0,0 +1,400 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Resco Reports",
+    "description": "Resco Reports provides you with a highly customizable tool for generating reports and documents based on your Dataverse / Dynamics data. This connector makes it possible to use the Resco Mobile Report templates to automate document generation or any other back-office scenarios.",
+    "contact": {
+      "name": "Resco experts are ready to listen and help",
+      "url": "https://www.resco.net/contact-support/"
```

Comment:
@JuroJ Please add an "email" property in the contact section. Your support contact email.

Reply:
Done